keyboards/acheron/athena/gamma/info.json

```diff
@@ -0,0 +1,137 @@
+{
+    "keyboard_name": "Athena rev Gamma",
+    "usb": {
+        "pid": "0x6586",
+        "device_version": "0.0.1"
+    },
+    "matrix_pins": {
+        "rows": ["C14","C15","A3" ,"C10","A8" ,"C12","C6" ,"C9" ,"B14","C2" ,"C1" ,"C8" ,"B9" ],
+        "cols": ["C11","A2" ,"C3" ,"A0" ,"A1" ,"B10","B1" ,"B0" ,"C5" ,"C4" ,"A7" ,"A6" ,"A5" ,"A4" ,"A15"]
+    },
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "lto":       true,
+        "mousekey":  true,
+        "extrakey":  true,
+        "console":   false,
+        "backlight": true,
+        "rgblight":  true,
+        "audio":     false,
+        "encoder":   false,
+        "nkro":      true
+    },
+    "backlight": {
+        "pin": "C7",
+        "levels": 20,
+        "breathing": true
+    },
+    "ws2812": {
+        "pin": "B15"
+    },
+    "processor": "STM32F401",
+    "bootloader": "stm32-dfu",
+    "layouts": {
```

Comment:
based on corresponding `readme.md` image, would it be possible to add the following Community Layouts?:
- tkl_ansi
- tkl_ansi_split_bs_rshift
- tkl_ansi_tsangan
- tkl_ansi_tsangan_split_bs_rshift
- tkl_f13_ansi
- tkl_f13_ansi_split_bs_rshift
- tkl_f13_ansi_tsangan
- tkl_f13_ansi_tsangan_split_bs_rshift
- tkl_f13_iso
- tkl_f13_iso_split_bs_rshift
- tkl_f13_iso_tsangan
- tkl_f13_iso_tsangan_split_bs_rshift
- tkl_iso
- tkl_iso_split_bs_rshift
- tkl_iso_tsangan
- tkl_iso_tsangan_split_bs_rshift

Reply:
Honestly I'd much rather just supply a `matrix_diagram.md`. Where can I find a diagram template?

Reply:
> Honestly I'd much rather just supply a `matrix_diagram.md`. Where can I find a diagram template?

(here's)[https://github.com/qmk/qmk_firmware/blob/57944ee32f35de632545ca856f7e3e86df89821f/keyboards/acheron/athena/beta/matrix_diagram.md] `matrix_diagram.md` for Athena Beta from a PR of mine. Replace/change matrix co-ordinates for Gamma 😊

Reply:
Nice, thanks! A matrix diagram was added in [030ea6d](https://github.com/qmk/qmk_firmware/pull/22141/commits/030ea6dd4c417bf0dd45c52c1e8f6b7182da7edd).